src/main/scala/no/ndla/conceptapi/controller/InternController.scala

```diff
@@ -162,6 +162,14 @@ trait InternController {
       }
     }
 
+    post("/dump/concept/") {
+      val concept = extract[Concept](request.body)
+      concept match {
+        case Success(c) => Ok(draftConceptRepository.insert(c))
```

Comment:
Burde vi ikke både ha en for `draft-concept` og `concept` slik som vi har over for å kunne migrere mellom miljøene?

Reply:
Usikker på om vi treng det. Alle artikler som skal importeres må manuelt publiseres fra draft til article og tenkte at det samme kan gjelde for forklaringer. Men endepunktet bør stemme med kvar det lagres, så det skal eg endre.